README.md

```diff
@@ -48,7 +48,7 @@ A Docker image can be built, then run by doing the following:
 ## API overview
 > For full documentation of the API, you can find the swagger page hosted at the root url. See [Installing](#Installing) to run the application and browse the docs.
 
-Currently the server just has one endpoint: `/bake`. This endpoint accepts a POST request with the following body:
+The most important endpoint is `/bake`. This endpoint accepts a POST request with the following body:
```

Comment:
Could you make this part of the README a list of endpoints with anchor tags to their respective parts of the documentation please?